actions/pr.yaml

```diff
@@ -50,9 +50,9 @@ jobs:
       - name: run python tests
         run: |
           pip3 install pytest
-          cd src
-          pip3 install -r apiserver/requirements.txt
-          pytest
+          cd src/apiserver
+          pip3 install -r requirements.txt
+          python -m pytest
```

Comment:
Why `python` and not `python3`? Is it intentional?

Reply:
Because setup-python version makes sure that 'python' has requested version (3.8 in our setup).